src/main/java/com/teamknp/hotel/domain/InvoiceInfo.java

```diff
@@ -0,0 +1,18 @@
+package com.teamknp.hotel.domain;
+
+import lombok.Getter;
+
+import java.math.BigDecimal;
+
+@Getter
+public class InvoiceInfo {
+    private final BigDecimal reservationInvoice;
```

Comment:
Lepiej stwórz obiekt Invoice, który będzie zawierać wszystkie potrzebne obiekty, a nie tylko będzie pojemnikiem na wyliczone wartości.

Reply:
Invoice czyli również wszystkie dokonane płatności?
Fajny pomysł. Zrobię to w drugim PR, ale póki co już zrobiłem przymiarkę.

Reply:
Tak.